File: devops-scm-api/src/main/java/com/tencent/devops/scm/api/pojo/PullRequestInput.java
package com.tencent.devops.scm.api.pojo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PullRequestInput {

    private String title;
    private String body;
    private String sourceBranch;
    private String targetBranch;
    private Object sourceRepo;
    private Object targetRepo;
}

File: devops-scm-api/src/main/kotlin/com/tencent/devops/scm/api/pojo/PullRequestInput.kt
package com.tencent.devops.scm.api.pojo

data class PullRequestInput(
    // PR/MR 标题
    val title: String,
    // PR/MR 正文内容
    val body: String? = null,
    // 源分支
    val sourceBranch: String,
    // 目标分支
    val targetBranch: String,
    // 源仓库
    val sourceRepo: Any? = null,
    // 目标仓库
    val targetRepo: Any? = null
)
